config/locales/en.yml

```diff
@@ -3842,32 +3842,25 @@ en:
               <li>A residence on which Payments-In-Lieu-of-Tax (P.I.L.O.T) payments were made. (If you think your building may have been subject to P.I.L.O.T payments, you can contact your building owner or the municipal tax office to confirm.)</li>
             </ul>
           helper_heading: How do I know if the home I rented was subject to property taxes?
+          instructions_html: "<p>Many people are surprised by this, but New Jersey has a law on the books that gives you a  property tax deduction or credit as a renter.</p> \n<p><strong>We will walk you through it.</strong></p>\n"
```

Comment:
[pebble] There's no need for the `\n`s if we're using paragraph tags to break lines